docs/examples/other_index.py

```diff
@@ -1,9 +1,22 @@
 doc(title="Other",
     underline_char="=",
     entries=[
-        "general/multigpu.ipynb",
-        "custom_operations/index.py",
-        "advanced/serialization.ipynb",
-        "legacy_getting_started.ipynb",
-        "general/debug_mode.ipynb",
+        doc_entry("general/multigpu.ipynb", [
+            op_reference("fn.readers.file", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.caffe", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.caffe2", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.coco", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.mxnet", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.nemo_asr", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.numpy", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.sequence", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.tfrecord", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.video", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.video_resize", "Reading the data in the multi-GPU setup."),
+            op_reference("fn.readers.webdataset", "Reading the data in the multi-GPU setup."),
+        ]),
+        doc_entry("custom_operations/index.py"),
+        doc_entry("advanced/serialization.ipynb"),
+        doc_entry("legacy_getting_started.ipynb"),
+        doc_entry("general/debug_mode.ipynb"),
```

Comment:
I don't think there is need to wrap those into doc_entry, but it doesn't matter much.